apps/condo/lang/en/en.json

```diff
@@ -536,6 +536,9 @@
   "pages.condo.ticket.id.CloseOrReturn": "Close or return",
   "pages.condo.ticket.id.TicketInWork": "the ticket to work?",
   "pages.condo.ticket.id.DownloadExcelLabel": "Download file",
+  "pages.condo.ticket.id.CancelModal.title": "Want to cancel a ticket?",
+  "pages.condo.ticket.id.CancelModal.cancelButtonLabel": "Cancel ticket",
+  "pages.condo.ticket.id.CancelModal.content": "This is the final status of the ticket, it will no longer be possible to transfer it to work or execute it. Indicate the reason for canceling the ticket in the comments so that other employees can see the information.",
```

Comment:
```suggestion
  "pages.condo.ticket.id.CancelModal.content": "This is the final status of the ticket, it will be impossible to take it for execution. Indicate the reason for canceling the ticket in the comments so that other employees can see the information.",
```